walkthrough/utils.py

```diff
@@ -17,6 +17,8 @@
 
 DAG_WALKTHROUGH_PATH = DAG_DIR / "walkthrough.yml"
 WALDEN_INGEST_DIR = Path(walden.__file__).parent.parent.parent / "ingests"
+DATASET_POPULATION_URI = "data://garden/demography/2023-03-31/population"
```

Comment:
Instead of hardcoding this specific version of the population dataset, it would be good to fetch the latest version. The optimal solution would be to look in the dag, but just for simplicity you could do something like:
```
from etl.paths import STEP_DIR
sorted((STEP_DIR / "data/garden/demography").glob("*/population/"))[-1].as_posix().split("/")[-2]
```
To get the latest version.

Reply:
Oh, ー sweet!

I was thinking about the risks of automatically getting the population URI this way.

Like, we are assuming that:

- we want to always use the latest population OMM (I think this is correct)
- the garden recipe for the latest available population dataset works (that should be the case)

We should continue to be extra careful when adding a new version of our population OMM (this occurs every 1-2 years, approx).